package.json

```diff
@@ -43,7 +43,8 @@
     }
   },
   "files": [
-    "lib"
+    "lib",
+    "src"
```

Comment:
Is there a reason that `src` should be distributed with the package?

Reply:
I believe this can be removed and an artifact of trying to add version command which is or will be addressed in a separate PR.